src/bric.c

```diff
@@ -112,127 +55,6 @@ int numbers_only(const char *s)
-                                                        case 'H': return HOME_KEY;
-                                                        case 'F': return END_KEY;
-                                                }
-                                        }
-                                }
-
-                                // ESC 0 sequences
-                                else if(seq[0] == 'O') {
-                                        switch (seq[1]) {
-                                                case 'H': return HOME_KEY;
-                                                case 'F': return END_KEY;
-                                        }
-                                }
-                                break;
-
-                        default:
-                                return c;
-                }
-        }
-}
-*/
-
-/* Use the ESC [6n escape sequence to query the horizontal cursor position
- *  * and return it. On error -1 is returned, on success the position of the
- *   * cursor is stored at *rows and *cols and 0 is returned. */
-/*
-int get_cursor_pos(int ifd, int ofd, int *rows, int *columns)
-{
-        char buf[32];
-        unsigned int i = 0;
-
-        // report cursor location
-        if (write(ofd, "\x1b[6n", 4) != 4) return -1;
-
-        // read the response: ESC [ rows; columns R
-        while(i < sizeof(buf)-1) {
-            if( read(ifd, buf+i, 1) != 1) break;
-            if(buf[i] == 'R') break;
-            i++;
-        }
-        buf[i] = '\0';
-
-        // parse it
-        if (buf[0] != ESC || buf[1] != '[') return -1;
-        if (sscanf(buf+2,"%d;%d",rows,columns) != 2) return -1;
-        return 0;
-
-}
-*/
-
-
-/*
-// try to get the number of columns in the window
-int get_window_size(int ifd, int ofd, int *rows, int *columns)
-{
-        struct winsize ws;
-
-        if(ioctl(1, TIOCGWINSZ, &ws) == -1 || ws.ws_col == 0) {
-                // ioctl() failed so query the terminal itself
-                int original_row, original_column, retval;
-
-                // get the initail position to store for later
-                retval = get_cursor_pos(ifd, ofd, &original_row, &original_column);
-                if(retval == -1) goto failed;
-
-                // go to the right bottom margin and get position
-                if(write(ofd, "\x1b[999C\x1b[999B", 12) != 12) goto failed;
-                retval = get_cursor_pos(ifd, ofd, rows, columns);
-                if(retval == -1) goto failed;
-
-                // restore the cursor position
-                char seq[32];
-                snprintf(seq, 32, "\x1b[%d;%dH", original_row, original_column);
-                if(write(ofd, seq, strlen(seq)) == -1) {
-                        // cant recover the cursor pos ....
-                }
-                return 0;
-        } else {
-                *columns = ws.ws_col;
-                *rows = ws.ws_row;
-                return 0;
-        }
-failed:
-        return -1;
-}
-*/
-
```

Comment:
I've delete this unused code, 'cause its content by `handling.c` and `handling.h`